### DISCUSSION TITLE
The Loudness Wars. Just DON’T.

### DISCUSSION THREAD

**Discussion by Moonbase59** [AzuraCast/AzuraCast#7177] [General]
Please don’t involve in this:

![Cd_loudness_trend-the_beatles-something](https://github.com/AzuraCast/AzuraCast/assets/3706922/8269dfed-79a6-4e2d-b33f-644152ac9e30)  
_The Beatles – Something. CD "Remasters" over the years._

### How about a **-18 LUFS Radio Counter-Culture** instead?

- **No more clipping until it hurts.**
- **No more compression-to-death and listener fatigue.**
- No more "I _mp3gain_ my files to _93 dee-bee!_" (Without even knowing what a dB was.)
- No more "But I wanna be the LOUDEST station on the dial!" (Why? Got no better niche?)
- No more "What must I set to get _0 dB?_" (Won’t tell.)
- No more "Hey, how can I crank up my compressor even _more?_" (Whut? To finally reach 0.5 LRA?)

Music and Radio enthusiasts will _love you!_

### Make love, not (Loudness) War. :vulcan_salute:

**Comment by RM-FM** [comment]
@Moonbase59 Ah, I'm only feeling semi-attacked here... our LRA is at least 0.9 ;-)

**Reply by Moonbase59** [reply]
Well, it’s a question of what the target audience _wants_. My hopes are that listeners will eventually again honor the _best_ station on the dial, not only the _loudest_. And listen to _that_. :smiling_face_with_tear:

**Comment by gAlleb** [comment]
_But I like my shit to play LOUD, mom!_

**Reply by Moonbase59** [reply]
Okayyyy… So how ’bout telling people they actually _do_ have a volume knob on their stereo? :smile:
